src/catalog/src/builtin.rs

```diff
@@ -155,12 +154,25 @@ pub struct BuiltinSource {
 pub struct BuiltinView {
     pub name: &'static str,
     pub schema: &'static str,
+    pub column_defs: Option<&'static str>,
     pub sql: &'static str,
     /// Whether the object should only be queryable by superusers,
     /// only by superusers and support, or by anyone.
     pub sensitivity: DataSensitivity,
 }
 
+impl BuiltinView {
+    pub fn create_sql(&self) -> String {
```

Comment:
We'll have to update the `impl Fingerprint` to use this method. It's down on line 274

```Rust
impl Fingerprint for &BuiltinView {
    fn fingerprint(&self) -> String {
        self.sql.to_string()
    }
}
```

instead of `self.sql` it should be `self.create_sql()`.

Reply:
It's possible that it might also help with the migration error that we're seeing.

Reply:
Interesting! I should've looked more closely with the `Fingerprint` trait. Thank you

Reply:
I tried it out locally and it doesn't seem to help. The panic is happening in the builtin migration framework, which is responsible for migrating system views (and other objects) when they're changed between releases. We use the `Fingerprint` trait to determine when an object has been changed. As you can see from the trait implementation, even if the white-space is changed in the CREATE SQL statement for a view, it will be considered changed. This is probably the largest migration we've had in awhile and  I think something in the framework itself isn't working properly.